File: exercises/03_exercise_conditional_statements_advanced/05_journey.py
budget = float(input())
season = input()
spent_money = 0
type_of_vacation = " "
destination = " "
if budget <= 100:
    destination = "Bulgaria"
    if season == "summer":
        spent_money = budget * 0.3
        type_of_vacation = "Camp"
    elif season == "winter":
        spent_money = budget * 0.7
        type_of_vacation = "Hotel"
elif budget <= 1000:
    destination = "Balkans"
    if season == "summer":
        spent_money = budget * 0.4
        type_of_vacation = "Camp"
    elif season == "winter":
        spent_money = budget * 0.8
        type_of_vacation = "Hotel"
elif budget > 1000:
    destination = "Europe"
    spent_money = budget * 0.9
    type_of_vacation = "Hotel"

print(f"Somewhere in {destination}")
print(f"{type_of_vacation} - {spent_money:.2f}")

File: 05_JavaScriptBasics/03_MoreExercises/03_conditionalStatementsAdvanced/05_journey.js
function journey(input){
    let budget = Number(input[0]);
    let season = input[1];
    let priceJourney = 0;
    let accommodation = "";
    let destination = "";


    if (budget <= 100){
        switch (season){
            case "summer":
                priceJourney = budget * 0.30;
                accommodation = "Camp";
                destination = "Bulgaria";
                break;
            case "winter":
                priceJourney = budget * 0.70;
                accommodation = "Hotel";
                destination = "Bulgaria";
                break;
        } 
    } else if (budget <= 1000){
        switch (season){
            case "summer":
                priceJourney = budget * 0.40;
                accommodation = "Camp";
                destination = "Balkans";
                break;
            case "winter":
                priceJourney = budget * 0.80;
                accommodation = "Hotel";
                destination = "Balkans";
                break;
        }
    } else {
        priceJourney = budget * 0.90;
        accommodation = "Hotel";
        destination = "Europe";
    }
    console.log(`Somewhere in ${destination}`);
    console.log(`${accommodation} - ${priceJourney.toFixed(2)}`)
}

journey(["50", "summer"]);
journey(["75", "winter"]);
journey(["312", "summer"]);
journey(["678.53", "winter"]);
journey(["1500", "summer"]);
